test/e2e/components/vl-multi-select.js

```diff
@@ -1,6 +1,113 @@
-const { VlElement } = require('vl-ui-core').Test;
+const { VlSelect } = require('vl-ui-select').Test;
+const { Pill } = require('./pill');
+const { By } = require('selenium-webdriver');
+
+class VlMultiSelect extends VlSelect {
+
+    async _getRoot() {
+        return this.findElement(By.xpath('../..'));
+    }
+
+    async _getSelectList() {
+        const root = await this._getRoot();
+        return root.findElement(By.css('.vl-select__list > div'));
+    }
+
+    async _getSelectListItems() {
+        const selectList = await this._getSelectList();
+        return selectList.findElements(By.css('.vl-select__item'));
+    }
```

Comment:
dit zijn dan de "nogNietGeselecteerdeItems"? zouden we beter dat ook niet in de naamgeving duidelijk maken?

Reply:
Aangepast.